lib/diagnostic.rb

```diff
@@ -93,7 +93,7 @@
 # In a Ruby string, write how you would remove the last two elements from this
 # array
 
-Diagnostic.remove_from_array = 'your answer here'
+Diagnostic.remove_from_array = arr.delete(2, 3)
 
```

Comment:
You could totally use the delete method, but not like this. 
A more common way would be `arr.pop 2`
Try getting it to work with the delete method also if you want!
https://ruby-doc.org/core-2.4.1/Array.html#method-i-delete_if